admin/templates/configuration/etc/qserv-wmgr.cnf

```diff
@@ -22,7 +22,7 @@ SECRET_KEY = '{{WMGR_SECRET_KEY}}'
 # ======== application-specific parameters =========
 
 #  database connection parameters, use None for defaults
-DB_HOST = '127.0.0.1'
+DB_HOST = 'localhost'
```

Comment:
with `localhost`, mysql client perform a connection using mysql socket, and qsmaster has correct access rights when using the socket. 127.0.0.1 was not working for inter containers communication, because of incorrect access right, it seems network was used there...